build-wheelhouse/action.yml

```diff
@@ -1,86 +1,97 @@
-name: "Build the wheelhouse of a Python library."
-description: "Generate ZIP artifacts for the wheelhouse of a Python library."
+name: >
+  Build wheelhouse action
+
+description: >
+  Generates compressed artifacts for the wheelhouse of a Python library. The
+  wheelhouse contains all the necessary dependencies to install the project.
+  This allows for local installations and avoids the need to connect to the
+  online PyPI index.
 
 inputs:
+
+  # Required inputs
+
   library-name:
-    description: 'Name of the Python library.'
+    description: >
+      Name of the Python library. This is the name used when uploading the wheel
+      and source distribution as artifacts. The name should be the same one used
+      in the PyPI index.
     required: true
     type: string
+
   library-namespace:
-    description: 'Namespace of the Python library.'
+    description: >
+      Namespace of the Python library. The namespace is the path used when
+      importing the library in Python. The library requires to have a
+      ``__version__`` variable at its top ``__init__.py`` file. 
     required: true
     type: string
+
   operating-system:
-    description: 'Name of the operating system.'
+    description: >
+      Name of the operating system where the library is installed.
     required: true
     type: string
+
   python-version:
-    description: 'Desired Python version for build.'
+    description: >
+      Python version used for installing and running ``pre-commit``.
     required: true
     type: string
+
+  # Optional inputs
+
   use-python-cache:
-    description: 'Whether to use Python cache or not. Requires a pyproject.toml file.'
+    description: >
+      Whether to use the Python cache for installing previously downloaded
+      libraries. If ``true``, previously downloaded libraries are installed from the
+      Python cache. If ``false``, libraries are downloaded from the PyPI index.
     required: false
     default: true
     type: boolean
-  target:
-    description: 'Optional target to be passed for the build process.'
-    required: false
-    default: ''
-    type: string
```

Comment:
Why did we delete the target @jorgepiloto? This is needed for the PyAnsys metapackage for example... The wheelhouse builds differently depending on the specified target. I recommend we add it back.

Reply:
Sorry, my fault. Let me bring it back.